### ISSUE TITLE
QueryException: php artisan migrate

### ISSUE THREAD

**Issue by cheichi** [grokability/snipe-it#13110]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hello, 

I did a clean new installation of Snipe-IT on a new server today. Everything worked without any problems. However, at the last step of data transfer from the previous installation, I get the following error message:

php artisan migrate

Migrating: 2021_06_07_155436_add_company_id_indexes
   Illuminate\Database\QueryException

  SQLSTATE[42000]: Syntax error or access violation: 1061 Duplicate key name 'accessories_company_id_index' (SQL: alter table `accessories` add index `accessories_company_id_index`(`company_id`))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:712
    708▕         // If an exception occurs when attempting to run a query, we'll format the error
    709▕         // message to include the bindings with SQL, which will make this exception a
    710▕         // lot more helpful to the developer instead of just the database's errors.
    711▕         catch (Exception $e) {
  ➜ 712▕             throw new QueryException(
    713▕                 $query, $this->prepareBindings($bindings), $e
    714▕             );
    715▕         }
    716▕     }

      +9 vendor frames
  10  database/migrations/2021_06_07_155436_add_company_id_indexes.php:18
      Illuminate\Support\Facades\Facade::__callStatic()

      +22 vendor frames
  33  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()
`

### Reproduction steps

1. Import Backup v5.1.5 - build 6029 (master)
2. Run php artisan migrate

### Expected behavior

Migration without errors....

### Screenshots

![error](https://github.com/snipe/snipe-it/assets/135139441/b9c90736-2741-4320-8754-e14125166d43)

### Snipe-IT Version

v6.1.1-pre - build 10727 (master

### Operating System

Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-73-generic x86_64)

### Web Server

Apache/2.4.52 (Ubuntu)

### PHP Version

PHP 8.1.2-1ubuntu2.11 (cli) (built: Feb 22 2023 22:56:18)

### Operating System

Windows 11

### Browser

Edge/Firefox

**Comment by marcusmoore** [comment]
Hi @cheichi, if I'm reading that right you have a backup from v5.1.5, imported it into a fresh v6.1.1pre app, and then got the error after running `php artisan migrate`?

**Comment by cheichi** [comment]
Yes, that is correct. I worked with the following instructions:

https://snipe-it.readme.io/docs/upgrading
https://snipe-it.readme.io/docs/moving-snipe-it

<blockquote><img src="https://snipe-it.readme.io/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/upgrading">Upgrading</a></strong></div><div>🚧If you run into any issues upgrading...Please check the Common Issues &gt; Installation/Upgrading Issues section which provides solutions for commonly encountered installation / upgrading problems.📘Note:When available, please use the more detailed upgrade instructions for specific versions, avail...</div></blockquote>
<blockquote><img src="https://snipe-it.readme.io/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/moving-snipe-it">Moving Snipe-IT</a></strong></div><div>Moving/migrating Snipe-IT to a new server should be a pretty painless process. You really only have a few things to worry about: Your .envYour databaseYour uploaded filesYour OAuth keys The easiest way to handle this is to simply run a backup through your Snipe-IT admin panel at Admin &gt; Backups, ...</div></blockquote>

**Comment by cheichi** [comment]
Good news, I was able to solve the problem. As usual, it was not a system error. When importing the old data, only the records were imported and not the whole tables. For this reason, the tables were already in the "new" format. Therefore the migration script could be executed successfully.

So I only had to import the whole old database including the structure and the migration script ran successfully. 

Everything runs ! Best software !
